ReadMe.md

```diff
@@ -390,7 +390,7 @@ Display list of all operations from start of API server: `curl -s localhost:7171
 * Optional query argument `filter` could filter actions on server side.
 * Optional query argument `last` could filter show only last `XX` actions.
 
-## Storages
+## Storage
```

Comment:
Why Storage here?

Reply:
Because storage is uncountable https://www.oxfordlearnersdictionaries.com/us/definition/english/storage

It is like the word water. People say I can drink different kinds of water: still, sparkly, and distilled. Same with the storage. This application supports many types of storage: S3, Azure, disk etc.

I don't mind, if you like storages, I can revert this change.

Reply:
maybe `Storage types` ?

Reply:
I like this, will update `storages -> storage types` in this file.